tools/wptrunner/wptrunner/testloader.py

```diff
@@ -467,7 +467,8 @@ def __init__(self,
                  total_chunks=1,
                  chunk_number=1,
                  include_https=True,
-                 skip_timeout=False):
+                 skip_timeout=False,
+                 executor_classes={}):
```

Comment:
Mutable objects in default arguments are not great.

Reply:
Got it